Tools/YamlCreate.ps1

```diff
@@ -373,19 +397,24 @@ Function Read-WinGet-InstallerValues {
         }
     }
 
-    $title   = 'Another Installer'
-    $msg     = '[Optional] Do you want to create another installer?'
-    $options = '&Yes', '&No'
-    $default = 1  # 0=Yes, 1=No
-
-    $AnotherInstaller = $Host.UI.PromptForChoice($title, $msg, $options, $default)
+    Write-Host
+    Write-Host
+    Write-Host -ForegroundColor 'White' "Another Installer"
+    Write-Host "[Optional] Do you want to create another installer?"
+    Write-Host -ForegroundColor 'White' -NoNewline "[Y] Yes  "
+    Write-Host -ForegroundColor 'Yellow' -NoNewline '[N] No '
+    Write-Host -NoNewline "(default is 'N'): "
+    do {
+        $keyInfo = [Console]::ReadKey($false)
+    } until ($keyInfo.Key)
 
-    if ($AnotherInstaller -eq '0') {
-        Read-WinGet-InstallerValues
+    if ($keyInfo.Key -eq 'Y') {
+        Write-Host; Read-WinGet-InstallerManifest
```

Comment:
```suggestion
        Write-Host; Read-WinGet-InstallerValues
```
```
Another Installer
[Optional] Do you want to create another installer?
[Y] Yes  [N] No (default is 'N'): y


[Optional] Enter any File Extensions the application could support. For example: html, htm, url (Max 256)
FileExtensions:
```
This is inverted, If Yes loop back

Reply:
You mean the issue is fixed now?

Reply:
This suggestion is still Inverted

My other issue seems to be solved, not sure where quhxl got looped since he didn't provide log

Reply:
```
Another Installer
[Optional] Do you want to create another installer?
[Y] Yes  [N] No (default is 'N'): n

[Optional] Enter any File Extensions the application could support. For example: html, htm, url (Max 256)
FileExtensions:
```
Nvm

Reply:
the last 2 commits did indeed solve the issue

Reply:
I have committed changes.